scanner/matcher/matcher.go

```diff
@@ -26,20 +39,22 @@ import (
 )
 
 // matcherNames specifies the ClairCore matchers to use.
+//
+// Note: Do NOT hardcode the names. It's very easy to mess up...
 var matcherNames = []string{
-	"alpine-matcher",
-	"aws-matcher",
-	"debian-matcher",
-	"gobin",
-	"java-maven",
-	"oracle",
-	"photon",
-	"python",
-	"rhel-container-matcher",
-	"rhel",
-	"ruby-gem",
-	"suse",
-	"ubuntu-matcher",
+	(*alpine.Matcher)(nil).Name(),
+	(*aws.Matcher)(nil).Name(),
+	(*debian.Matcher)(nil).Name(),
+	(*gobin.Matcher)(nil).Name(),
+	(*java.Matcher)(nil).Name(),
+	(*oracle.Matcher)(nil).Name(),
+	(*photon.Matcher)(nil).Name(),
+	(*python.Matcher)(nil).Name(),
+	rhcc.Matcher.Name(),
+	(*rhel.Matcher)(nil).Name(),
```

Comment:
RHEL matcher is the VEX matcher, right?

Reply:
yes, though the matcher's name did not change. It's still `rhel`. Just the functionality was changed to work for VEX data